great_ape_safe/ape_api/balancer.py

```diff
@@ -136,17 +136,12 @@ def find_best_pool_for_swap(self, asset_in, asset_out):
 
     def order_tokens(self, underlyings, mantissas=None):
         # helper function to order tokens/amounts numerically
+        underlyings_checked = [Web3.toChecksumAddress(x.lower()) for x in underlyings]
         if mantissas:
-            tokens = dict(zip([x.lower() for x in underlyings], mantissas))
+            tokens = dict(zip(underlyings_checked, mantissas))
             sorted_tokens = dict(sorted(tokens.items()))
-            sorted_underlyings, sorted_mantissas = zip(*sorted_tokens.items())
-            underlyings_checksummed = [
-                Web3.toChecksumAddress(x) for x in sorted_underlyings
-            ]
-            return underlyings_checksummed, sorted_mantissas
-        else:
-            sorted_tokens = sorted([x.lower() for x in underlyings])
-            return [Web3.toChecksumAddress(x) for x in sorted_tokens]
+            return list(sorted_tokens.keys()), list(sorted_tokens.values())
```

Comment:
addresses should be in lowercase when sorting is performed, [ref](https://dev.balancer.fi/guides/swaps/batch-swaps#code-walkthrough)

ex: 
```py
>>> underlyings = [weth.address, bal.address]
>>> sorted([x.lower() for x in underlyings])
["0xba100000625a3754423978a60c9317c58a424e3d", "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"]

>>> sorted([web3.toChecksumAddress(x) for x in underlyings])
["0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2", "0xba100000625a3754423978a60c9317c58a424e3D"]

Reply:
thanks, reverted to your version in 8481a11